tfhe/src/integer/gpu/server_key/radix/bitwise_op.rs

```diff
@@ -81,16 +81,9 @@ impl CudaServerKey {
         let shift_plaintext = u64::from(scalar) * delta;
 
         let scalar_vector = vec![shift_plaintext; ct_blocks];
-        let mut d_decomposed_scalar = CudaVec::<u64>::new_async(
-            ct.as_ref().d_blocks.lwe_ciphertext_count().0,
-            streams,
-            streams.gpu_indexes[0],
-        );
-        d_decomposed_scalar.copy_from_cpu_async(
-            scalar_vector.as_slice(),
-            streams,
-            streams.gpu_indexes[0],
-        );
+        let mut d_decomposed_scalar =
+            CudaVec::<u64>::new_async(ct.as_ref().d_blocks.lwe_ciphertext_count().0, streams, 0);
```

Comment:
Good catch. We should be accessing index 0 instead of `streams.gpu_indexes[0]` indeed. This actually fixes a possible bug.